src/rosdiscover/observer/ros2.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from roswire import AppInstance
+
+from .observer import Observer
+from .. import Config
+
+
+class ROS2Observer(Observer):
+
+    def __init__(self, app: AppInstance, config: Config):
```

Comment:
No need to define this constructor. This is already the implicit constructor.

Reply:
Actually, there seems to be a need to do this because the implicit constructor doesn't seem to get called (it gives a mypy error).